client/state/media/actions.js

```diff
@@ -6,21 +6,78 @@ import { castArray } from 'lodash';
 /**
  * Internal dependencies
  */
-import { MEDIA_DELETE, MEDIA_RECEIVE } from 'state/action-types';
+import {
+	MEDIA_DELETE,
+	MEDIA_RECEIVE,
+	MEDIA_REQUEST,
+	MEDIA_REQUEST_FAILURE,
+	MEDIA_REQUESTING } from 'state/action-types';
```

Comment:
I personally like trailing command as well. For the same reasons I like separate `var` statements instead of comma separating them.

```
var something
var somethingElse
```

But I'll follow whatever is preferred.